### ISSUE TITLE
Error building under jruby

### ISSUE THREAD

**Issue by kspurgin** [trln/argot-ruby#8]
```
kspurgin@L004904:~/trln/argot-ruby$ ruby -v
jruby 9.1.8.0-SNAPSHOT (2.3.1) 2017-01-20 152cf90 Java HotSpot(TM) 64-Bit Server VM 25.121-b13 on 1.8.0_121-b13 +jit [linux-x86_64]
kspurgin@L004904:~/trln/argot-ruby$ ls
argot.gemspec  CHANGELOG.md  Jarfile.lock  LICENSE   README.md   test
bin            Jarfile       lib           Rakefile  ROADMAP.md
kspurgin@L004904:~/trln/argot-ruby$ gem build argot.gemspec 
Invalid gemspec in [argot.gemspec]: missing class name (`org.noggit.JSONParser')
ERROR:  Error loading gemspec. Aborting.
```

Is there something else that needs to be installed first for JRuby? If so, can where to get/how to install this be documented? 

(successfully installed argot-ruby gem under MRI 2.3.3)

**Comment by adjam** [comment]
@kspurgin what happens if you run 

    $ gem install jbundler

and then try to build the gem again?

**Comment by kspurgin** [comment]
@adjam The above is what happens in that case. 

I got a different error previously and installing the jbundler gem fixed that. 

This is what happened next. I looked to see if there was a noggit or JSONParser gem, but it looks like noggit is a Java library(? - not up on the proper Java lingo), but it wasn't clear to me how to get what I needed or where to put it so that Jruby can use it. 

Thanks!

**Comment by adjam** [comment]
There isn't a lot of good documentation out there about the tools that let you bundle jars (Java ARchives) with gems, so I'm going to have to look into this further.  Until I have progress on that front, just use MRI.

**Comment by adjam** [comment]
Open PR #9 may address this.  You can git clone the issue-8 branch and see if that resolves the problem.  Should work with a

    $ bundle install

before attemping to build the gem.

**Comment by adjam** [comment]
@kspurgin I have this fixed, as far as I can tell.  It will be available when somebody merges the pull request, or you can use the branch for the time being.

**Comment by kspurgin** [comment]
Ok, thanks, @adjam 
I have been buried under other stuff but will give this another try when I have a chance. 
Was able to get the MRI version to work, but my understanding is this would likely be faster at scale.

**Comment by adjam** [comment]
@kspurgin at the moment, using JRuby is only likely to slow you down; the support is there so we can investigate whether it will provide benefits down the road, and these benefits are most likely to show up on the server side (in a nutshell, JRuby is only unquestionably 'better' at running multiple threads of execution concurrently, and multiple threads aren't always faster than a single thread).  

Until you run into a problem where MRI seems "too slow", I suggest sticking with that.

**Comment by queryluke** [comment]
fixed with PR #9

**Comment by kspurgin** [comment]
@adjam -- good to know. I'll probably not look at this again any time soon then!
